src/lib.rs

```diff
@@ -1776,19 +1900,18 @@ impl<
         mut backend: Backend,
         loader: &mut impl KeystoreLoader<L, Meta = Meta>,
     ) -> BoxFuture<'a, Result<Keystore<'a, Backend, L, Meta>, KeystoreError<L>>> {
-        let mut atomic_loader = AtomicStoreLoader::load(&loader.location(), "keystore").unwrap();
-        let mut storage = AtomicKeystoreStorage::new(loader, &mut atomic_loader, 1024).unwrap();
-        let mut atomic_store = AtomicStore::open(atomic_loader).unwrap();
+        let (mut atomic_store, mut storage, mut assets) = Self::create_stores(loader).unwrap();
         Box::pin(async move {
             let state = if storage.exists() {
                 storage.load().await?
             } else {
-                let state = backend.create().await?;
+                let state: KeystoreState<'a, L> = backend.create().await?;
```

Comment:
For my own curiosity any idea why it made you add the type annotation here?

Reply:
I've no idea! Probably for fixing a build error that's no longer needed. I'll remove it.